File: src/main/java/com/alan/Hello.java
package com.alan;

import jdk.jfr.internal.settings.PeriodSetting;

//類別
public class Hello {
    //程式主要執行入口
    public static void main(String[] args) {    //縮寫：psvm+TAB
        //方法一
        //new Person().hello();

        //方法二
        Person p=new Person();
        p.weight=66.5f;
        p.height=1.75f;
        System.out.println( p.bmi() );

        //方法三
        Person p2=new Person(66.5f,1.75f);
        System.out.println(p2.bmi());
    }
}

File: src/main/kotlin/com/alan/Hello.kt
package com.alan

import java.util.*

//不用使用類別呼叫，直接fun main即可使用 或 main+TAB
fun main() {
    //方法一
    //Human().hello()

    //方法二
    //val h = Human();//不需要使用new即可建立物件
    //h.hello();

    //方法三
    val h = Human(66.5f,1.75f)
    println(h.bmi())
}

